src/App.js

```diff
@@ -6,11 +7,26 @@ import Home from './pages/Home/Home';
 import './App.scss';
 
 function App() {
+  const [token, setToken] = useState("")
+
+  useEffect(() => {
+    const hash = window.location.hash
+    let token = localStorage.getItem("token")
+
+    if (!token && hash) {
+        token = hash.substring(1).split("&").find(elem => elem.startsWith("access_token")).split("=")[1]
+        window.location.hash = ""
+        localStorage.setItem("token", token)
+    }
+
+     setToken(token)
```

Comment:
Can we add this to a function inside of this and call the function in the useEffect?